docs/embedding/sdk/questions.md

```diff
@@ -139,26 +139,169 @@ To customize the layout, use namespaced components within the `InteractiveQuesti
-| `Filter`                   | A set of badges for adding, viewing, and editing the filters of the current question                                         |
-| `FilterPicker`             | Picker for adding a new filter to the question                                                                               |
-| `FilterDropdown`           | A button which has a dropdown for adding, viewing, and editing the filters of the current question                           |
-| `ResetButton`              | The button used to reset the question after the question has been modified with filters/aggregations/etc                     |
-| `Title`                    | The question's title                                                                                                         |
-| `SaveButton`               | Button for saving the question.                                                                                              |
-| `Breakout`                 | A set of badges for adding, viewing, and editing the groupings of the current question                                       |
-| `BreakoutDropdown`         | A button which has a dropdown for adding, viewing, and editing the groupings of the current question                         |
-| `Summarize`                | A set of badges for adding, viewing, and editing the groupings of the current question                                       |
-| `SummarizeDropdown`        | A button which has a dropdown for adding, viewing, and editing the groupings of the current question                         |
-| `Notebook`                 | The Notebook editor that allows for more filter, aggregation, and custom steps                                               |
-| `NotebookButton`           | The button used in the default layout to open the Notebook editor. You can replace this button with your own implementation. |
-| `QuestionVisualization`    | The chart visualization for the question                                                                                     |
-| `QuestionSettings`         | The settings for the current visualization                                                                                   |
-| `QuestionSettingsDropdown` | A settings button that contains the `QuestionSettings` component in the dropdown                                             |
-| `ChartTypeDropdown`        | A dropdown containing the possible visualization options for the current question                                            |
+#### `InteractiveQuestion.BackButton`
+> A navigation button that returns to the previous view. Only renders when `onNavigateBack` is available in context.
 
+Uses [Mantine ActionIcon props](https://v6.mantine.dev/core/action-icon/) under the hood, as well as:
+| Prop | Type | Description |
+|------|------|-------------|
+| className | string | |
```

Comment:
I think we should either add a short description for all `className`s and `style`s or just remove the description column from them since they look odd imo being blank like this.

![image](https://github.com/user-attachments/assets/a718ec12-ad36-47ba-997e-982224b11eca)

I see some components have other props other than these 2 in the same table as well. I think giving them short and concise description should do the trick.